scheduler/src/cook/test/testutil.clj

```diff
@@ -302,6 +305,31 @@
                                  mesos-start-time (assoc :instance/mesos-start-time mesos-start-time))])]
     (d/resolve-tempid (db conn) (:tempids val) id)))
 
+(defn make-task-request [job-ent]
+  (let [considerable->task-id (plumbing.core/map-from-keys (fn [_] (str (d/squuid)))
```

Comment:
clojuredoc this and the other 2 functions please.

Reply:
Done